state/event_table_test.go

```diff
@@ -1138,15 +1138,16 @@ func TestEventTable_SelectLatestEventsBetween_MissingPrevious(t *testing.T) {
 	}
 	defer txn.Rollback()
 	roomID := fmt.Sprintf("!%s", t.Name())
+	// Event IDs ending with `-gap` have MissingPrevious: true.
 	events := []Event{
 		{ID: "A", MissingPrevious: false},
 		{ID: "B", MissingPrevious: false},
 		{ID: "C", MissingPrevious: false},
-		{ID: "D", MissingPrevious: true},
+		{ID: "D-gap", MissingPrevious: true},
 		{ID: "E", MissingPrevious: false},
 		{ID: "F", MissingPrevious: false},
-		{ID: "G", MissingPrevious: true},
-		{ID: "H", MissingPrevious: true},
+		{ID: "G-gap", MissingPrevious: true},
+		{ID: "H-gap", MissingPrevious: true},
```

Comment:
I think I'm just having a lot of mental block here because I see A B C D and instinctively think there is no gap. When you then put `H-gap` I think "uhh before or after"? Can we +2 letters to hint at the gap maybe? E.g A B C F-gap?

Reply:
Hmm. I think rename these to e.g. `chunkX-eventY` , so that gaps only appear when we change chunks.